train.sh

```diff
@@ -3,10 +3,20 @@
 . config
 . lib.sh
 
-check_config "IAM_ROLE OWNER DATASET_OWNER IMAGE PROJECT"
-parse_args "$0" "experiment dataset dataset_owner=${DATASET_OWNER} model task=${TRAIN_TASK_NAME} load_from=None" "$@"
+parse_args "$0" "experiment dataset dataset_owner=${DATASET_OWNER} model task=${TRAIN_TASK_NAME} load_from=None num_gpus=1" "$@"
 shift $n
 
+if [ "$num_gpus" = "1" ]; then
+  GPU_NUM=1
+	GPU_TYPE="p3.2xlarge"
+else
+  GPU_NUM=$num_gpus
+	GPU_TYPE="p3.8xlarge"
+fi
+
+check_config "IAM_ROLE OWNER DATASET_OWNER IMAGE PROJECT GPU_NUM GPU_TYPE"
```

Comment:
GPU_NUM and GPU_TYPE are not configuration variables, so this does not make sense.